<launch>
   <!-- Launch temperature node -->
   <node name="temperature" pkg="temperature"  type="temperature.py" output="screen"/>

   <!-- Launch status led node -->
   <node name="status_led" pkg="status_led"  type="statusled.py" output="screen"/>

   <!-- Launch imu node -->
   <node name="imu" pkg="imu"  type="imu.py" output="screen">
   	<param name="x_cal" value="0.0" />
   	<param name="y_cal" value="0.0" />
   	<param name="z_cal" value="0.0" />
   </node>
   
   <!-- Launch rosbag -->
   <node pkg="rosbag" type="record" name="rosbag_record_robot"
            args="/imu/data_raw /imu/mag /temp"
    />

</launch>

